common/djangoapps/student/roles.py

```diff
@@ -130,11 +130,17 @@ def __init__(self, role_name, org='', course_key=None):
         self.course_key = course_key
         self._role_name = role_name
 
-    def has_user(self, user):
+    def has_user(self, user, check_user_activation=True):
         """
-        Return whether the supplied django user has access to this role.
+        Check if the supplied django user has access to this role.
+
+        Arguments:
+            user: user to check against access to role
+            check_user_activation: do not check user activation while removing user
```

Comment:
I would explain it like this. 
`check_user_activation (Boolean): Indicating whether or not we need to check user activation while checking user roles`